lib/ansible/modules/packaging/os/redhat_repositories.py

```diff
@@ -24,56 +24,58 @@
 module: redhat_repository
 short_description: Manage repositories with RHSM using the C(subscription-manager) command
 description:
-    - Manage repositories with the Red Hat Subscription Management entitlement platform using the C(subscription-manager) command
+  - Manage repositories with the Red Hat Subscription Management entitlement platform using the C(subscription-manager) command
```

Comment:
I see that some blocks are wrapped at the 80 characters but for example this line is not. It would be nice to have all wrapped the same way.